File: Go/Empregado.go
package listaRicardo

import "fmt"

type Empregado struct {
	Nome    string
	Cargo   string
	Salario float64
}

func (e Empregado) InfoEmpregado() {
	fmt.Printf("Nome do funcionário: %s\n", e.Nome)
	fmt.Printf("Cargo do funcionário: %s\n", e.Cargo)
	fmt.Printf("Salário do funcionário: R$ %.2f\n", e.Salario)
}

File: Java/Empregado.java
public class Empregado {
    private String nome;
    private String cargo;
    private double salario;

    public Empregado(String nome,String cargo,double salario){
        this.nome = nome;
        this.cargo = cargo;
        this.salario = salario;
    }

    public void infoEmpregado(){
        System.out.println("Nome do funcionário: "+ this.nome);
        System.out.println("Cargo do funcionário: "+ this.cargo);
        System.out.println("Salário do funcionário: "+ this.salario);
    }
}



